.dockerignore

```diff
@@ -0,0 +1,8 @@
+\.git
+Dockerfile
+\.gitignore
+\.prettierrc
+LICENSE
+\.next
+node_modules
+README\.md
```

Comment:
@Entkenntnis Can you check whether this contains all files which are not necessary for building the frontend with nextjs?